__tests__/app.test.js

```diff
@@ -162,6 +162,97 @@ describe("news-api", () => {
             })
         })  
     })
+    describe("POST requests", () => {
+        describe("POST /api/articles/:article_id/comments", () => {
+            test("responds with status code 201 and the posted object in expected format", () => {
+                return request(app)
+                .post("/api/articles/1/comments")
+                .send({"username": "butter_bridge", "body": "test"})
+                .expect(201)
+                .then(({body}) => {
+                    const { comment } = body
+                    expect(comment.comment_id).toEqual(expect.any(Number))
+                    expect(comment.votes).toEqual(expect.any(Number))
+                    expect(comment.created_at).toEqual(expect.any(String))
+                    expect(comment.author).toEqual(expect.any(String))
+                    expect(comment.body).toEqual(expect.any(String))
+                    expect(comment.article_id).toEqual(expect.any(Number))
+                })
+            })
+            test("posts the new comment to the database", () => {
+                const sentObj= {"username": "icellusedkars", "body": "test2"}
+
+                return request(app)
+                .post("/api/articles/1/comments")
+                .send(sentObj)
+                .expect(201)
+                .then(() => {
+                    return request(app)
+                    .get("/api/articles/1/comments")
+                    .expect(200)
+                    .then(({ body }) => {
+                        const { comments } = body
+                        const { username: sentUsername, body: sentBody} = sentObj
+                        const includesComment = comments.some((comment) => {
+                            const { author, body } = comment
+                            return author === sentUsername && body === sentBody
+                        })
+                        expect(includesComment).toBe(true)
+                    })
+                })
+            })
```

Comment:
By making the changes to the above test would mean that this test is no longer needed, and it would be done in a more readable way.